### ISSUE TITLE
Octave d' has a ", " prior to number in supervisor.xlsx

### ISSUE THREAD

**Issue by nohelix** [Auerbach-Lab/Behavior-autoanalysis#36]
Not sure where its coming from because it looks ok when it generates the table.
[supervisor.xlsx](https://github.com/Auerbach-Lab/Behavior-autoanalysis/files/10714807/supervisor.xlsx)

**Comment by nohelix** [comment]
The supervisor.xlsx is put a list in to the d' column. However, the table generated (r) at line 679 is correct:
`task     detail date   file_~1  weight trial~2 hit_p~3 FA_pe~4   Rxn mean_~5 warni~6 comme~7 Spacer1 dprim~8 Spacer2 ...16 ...17 ...18 ...19
  <chr>    <chr>  <chr>  <chr>     <dbl>   <dbl>   <dbl>   <dbl> <dbl>   <dbl> <list>  <chr>   <lgl>     <dbl> <lgl>   <lgl> <lgl> <lgl> <lgl>
1 Training Normal Overa~ Averag~ -0.0312     523   0.984  0.0227 0.432    1.28 <NULL>  NA      NA        NA    NA      NA    NA    NA    NA   
2 Training Normal 02/13~ 16kHz_~ -0.0338     504   0.989  0.0331 0.430    1.32 <list>  barely~ NA         4.03 NA      NA    NA    NA    NA   
3 Training Normal 02/11~ 16kHz_~ -0.0287     542   0.979  0.0123 0.434    1.25 <list>  good h~ NA         4.17 NA      NA    NA    NA    NA  `

Best guess is that the column is getting written to excel as a list instead of as numbers.
[supervisor.xlsx](https://github.com/Auerbach-Lab/Behavior-autoanalysis/files/10734282/supervisor.xlsx)

**Comment by nohelix** [comment]
Tibble hid the issue. 

d' was a list with only one value so Tibble claimed 'double' but it was actually a list with a single double. I only found out when I tried to write it out as a csv and write_csv and write.csv both ID'ed column 14 (d') as a list causing them to threw errors. Its now just unpacked from the list because it should always only be a double at (within octave training at least).

That openxlsx was trying to handle being given a non-flat file rather than dieing was causing the full list of d' s to be repeated each day.
